src/popup/components/TransactionList.vue

```diff
@@ -197,17 +232,30 @@ export default defineComponent({
 
     async function getLatest() {
       try {
-        await root.$store.dispatch('fetchTransactions', { limit: 10, recent: true });
+        await root.$store.dispatch('fetchTransactions', { limit: 10, recent: true, address: currentAddress.value });
       } finally {
         loading.value = false;
       }
     }
 
+    async function setMultisigTransaction(acc: IMultisigAccount | undefined) {
+      if (acc) {
+        multisigTransaction.value = {
+          ...await fetchActiveMultisigTx(acc.contractId),
+        };
+      }
+    }
+
     watch(displayMode, () => {
       checkLoadMore();
     });
 
-    onMounted(() => {
+    onMounted(async () => {
+      if (isMultisigDashboard.value) {
+        loading.value = true;
+        await watchUntilTruthy(() => activeMultisigAccount.value);
```

Comment:
I suggest to avoid using the `watchUntilTruthy` function as I think we need to get rid of it. Also this line blocks running the code that is placed after the `if`.

Maybe using Vue watcher before the onMounted would be better?

```
watch(activeMultisigAccount, (acc) => {
  ...
});
```

Reply:
It should wait for the account as it needs it's address in loadMore function